### PR TITLE
Sonir/cart

### PR DESCRIPTION
**Pull request by SonirAster**
Created Cart Modal.
Updated Notifications Templates using Cancel Button Template.

### CODE REVIEW

File: src/components/Cancel/Cancel.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        case 'light': 
+            return <StyledCancelLight />;
+        case 'dark': 
+            return <StyledCancelDark />;
+        default: 
+            return  <StyledCancelDark />;
+    }
+}
+
+export default Cancel;
```
Comment (whatislove23) [inline src/components/Cancel/Cancel.jsx]: pls open Button component in dev and look how you can implement this switch case statement 
Like this:
image

File: src/components/Cart/CartModal/CartModal.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    if (props.isEmpty) cartModal = <CartEmpty />
+    else cartModal = <CartFilled />
+
+    return (
+        <StyledCartModal>
+            <StyledCartModalHeader>
+                <h2>Your cart</h2>
+                <Cancel cancelStyle='dark' />
+            </StyledCartModalHeader>
+            {cartModal}
```
Comment (whatislove23) [inline src/components/Cart/CartModal/CartModal.jsx:62]: I meant shorter way
 "{props.isEmpty?
 CartEmpty :
 CartFilled }"

**Review by whatislove23 — COMMENTED** [review]
Pls check my recommendation

File: src/components/Cart/CartModal/CartModal.jsx (hunk 2 above)
Comment (SonirAster) [reply src/components/Cart/CartModal/CartModal.jsx:62]: What? This property was written only to define should the modal be empty or show the list of products in it.

File: src/components/Cancel/Cancel.jsx (hunk 1 above)
Comment (SonirAster) [reply src/components/Cancel/Cancel.jsx]: No problem. Done!

File: src/components/Cart/CartModal/CartModal.jsx (hunk 2 above)
Comment (whatislove23) [reply src/components/Cart/CartModal/CartModal.jsx:62]: use ternary operator

Comment (SonirAster) [reply src/components/Cart/CartModal/CartModal.jsx:62]: Ready.

**Pull request merged**